docs/index.md

```diff
@@ -1295,6 +1298,69 @@ The "HTML" reporter is what you see when running Mocha in the browser.  It looks
 
 [Mochawesome](https://www.npmjs.com/package/mochawesome) is a great alternative to the default HTML reporter.
 
+## Desktop Notification Support
+
+Desktop notifications allow asynchronous communication of events without
+forcing you to react to a notification immediately. Their appearance
+and specific functionality vary across platforms. They typically disappear
+automatically after a short delay, but their content is often stored in some
+manner that allows you to access past notifications.
+
+[Growl][] was an early notification system implementation for OS X and Windows,
```

Comment:
I think you can even omit `[]`

Reply:
Took some time to find documentation on [shortcut reference link](https://github.github.com/gfm/#shortcut-reference-link), but prefer the [collapsed reference link](https://github.github.com/gfm/#collapsed-reference-link) style; . Didn't know the former even existed, and I've been writing markdown for years. As I sometimes use square brackets in my text, I'm not sure I care for this _feature_.

Reply:
agree it's unexpected; I didn't discover it until I saw it being used in Mocha's CHANGELOG.  might be a peculiarity of GFM